downstream/assemblies/platform/assembly-installing-controller-operator-external-db.adoc

```diff
@@ -8,48 +8,49 @@ ifdef::context[:parent-context: {context}]
 
 
 [id="installing-contr-operator-external"]
-= Installing {ControllerName} on {OCP} with an external database and routed web traffic
+= Installing {ControllerName} on {OCP} with an external database
 
 
 :context: installing-contr-operator-external
 
 
 [role="_abstract"]
-You can use these instructions to install the {HubName} operator on {OCP} with an external PostgreSQL 12 database and route, as well as specify customer resources.
+You can use these instructions to install the {ControllerName} operator on {OCP} with an external PostgreSQL 12 database and route, as well as specify customer resources.
```

Comment:
Do you think we should keep using **{ProductName} operator**, or the attributes defined specifically for the operator? 
**{ProductName} operator** is cleaner. 
The only advantage to using the defined operator names is a possible future name change for the operators.

Reply:
I'm honestly not sure. I see it working either way.

@cbudz what do you think? We can either go with {ProductName} Operator or create its own {ProductOperator}

Reply:
@TarikFD lets go with {PlatformOperator} just in case they change it.